routes/payout.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const { rest } = require('lodash');
+const router = express.Router();
+const User = require("../models/user.model");
+
+const { baseURL } = require('../config/index');
+
+const stripeSecretKey = process.env.STRIPE_SECRET_KEY
+const stripe = require('stripe')(stripeSecretKey);
+
+router.post('/setup', async(req, res) => {
+
+  const { email } = req.body;
+
+  try{
+      const account = await stripe.accounts.create({
+          type: 'express',
+          country: 'US',
+        });
+
+      const accountLink = await stripe.accountLinks.create({
+      account: account.id,
+      refresh_url: `${baseURL}/myAccount`,
+      return_url: `${baseURL}/myAccount`,
+      type: 'account_onboarding',
+      });
+
+      const redirectURL = accountLink['url'];
+
+      const user = await User.findOne({ email });
+      user.stripeId = account.id;
+      await user.save();
+
+      res.status(200).json({
+        link: redirectURL,
+        id: account.id
+      })
+    } catch (e){
+      console.log(e)
+      res.status(500).json({
+        'error': 'Stripe accountLink failed'
+      })
+    } 
+}
+
+);
+
+router.post('/express', async(req, res) => {
```

Comment:
What does this route do? Could we add some comments to document both the routes in `payout.js`